src/algorithms/math/vector/README.md

```diff
@@ -0,0 +1,11 @@
+#Euclidean vector
+
```

Comment:
Please make PR to be focused. Don't commit Vector along with Complex Numbers. Please leave only Complex Numbers here in this PR.